### PR TITLE
feat(@fremtind/jkl-validators-util): adds validator for dog-id to val…

### PR DESCRIPTION
**Pull request by Mikaila94**
…idator-utils

affects: @fremtind/jkl-validators-util

Adds validator for dog-id to validator-utils

ISSUES CLOSED: #1508

## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/validators-util/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@fremtind/jkl-validators-util",
-    "version": "1.2.19",
+    "version": "1.3.0",
```
Comment (lfbergee) [inline packages/validators-util/package.json:3]: Ikke nødnvendig å oppdatere versjonsnummer. Det gjøres automatisk ved release basert på commitmeldingene.

**Review by lfbergee — APPROVED** [review]

**Comment by Mikaila94** [comment]
**NB: Bytt base-branch til master når #1536 er merget inn**

**Comment by lfbergee** [comment]
> **NB: Bytt base-branch til master når #1536 er merget inn**

Det skjer nok automagisk :)

**Comment by Mikaila94** [comment]
rebased on master

**Comment by lfbergee** [comment]
Det er noe rusk i naming av dokumentasjonsfilene dine:

```
@fremtind/portal: Can't resolve './IsValidDogIdExample' in '/home/runner/work/jokul/jokul/packages/validators-util/documentation'
@fremtind/portal: If you're trying to use a package make sure that './IsValidDogIdExample' is installed. If you're trying to use a local file make sure that the path is correct.
```

Ser ut til at du har liten forbokstav isteden for stor

**Comment by Mikaila94** [comment]
Renames isValidDogIdExample -> IsValidDogIdExample

**Review by lfbergee — APPROVED** [review]

**Pull request merged**